### DISCUSSION TITLE
Mask input field

### DISCUSSION THREAD

**Discussion by SangeetAgarwal** [remix-run/remix#6440] [Q&A] [answered]
How would implement an input with a mask?

**Comment by machour** [comment, marked as answer]
By using a React library, or implementing your own component: https://dev.to/juanmanuelcrego/input-mask-in-react-without-libraries-5akf

PS: Let's try to keep the discussions about Remix stuff :)

**Reply by SangeetAgarwal** [reply]
Thanks & my bad, server rendered app frameworks are a bit new to me.
